react/features/stream-effects/presenter/index.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import JitsiStreamPresenterEffect from './JitsiStreamPresenterEffect';
+
+/**
+ * Creates a new instance of JitsiStreamPresenterEffect.
+ *
+ * @param {MediaStream} stream - The video stream which will be used for
+ * creating the presenter effect.
+ * @returns {Promise<JitsiStreamPresenterEffect>}
+ */
+export function createPresenterEffect(stream: MediaStream) {
+    if (!MediaStreamTrack.prototype.getSettings
+        && !MediaStreamTrack.prototype.getConstraints) {
+        return Promise.reject(new Error('JitsiStreamPresenterEffect not '
```

Comment:
doesn't this fit in a single line?

Reply:
This line is also longer than 80 characters.